quicksort/QuickSort.java

```diff
@@ -0,0 +1,36 @@
+public class QuickSort {
+
+    static int partition(int[] a, int first, int last) {
+        int pivot = a[last];
+        while(first<last) {
```

Comment:
Please Add a space between bracket and `while` so as to make it look different from function's (method's). Also add space before and after any binary operator